.travis.yml

```diff
@@ -13,24 +48,16 @@ env:
   global:
     - TEST_PHP_ARGS="-q -s output.txt -g XFAIL,FAIL,BORK,WARN,LEAK,SKIP -x --show-diff"
     - REPORT_EXIT_STATUS=1
-    - MONGODB_ORCHESTRATION="http://localhost:8889"
-    - MONGODB_ORCHESTRATION_REPO="git+git://github.com/mongodb/mongo-orchestration@master"
-    - MONGODB_ORCHESTRATION_PRESETS_ROOT=`pwd`
-    - secure: "Dofd9NvXIKEat1PMiia3RQgNsJLaKtzZRf9rM4MsqFaBQPbJmVWPANJkayJPcK+l8pLxHyzzH7WqxML+SlBhBTHdJB++1woXxoVwm+sDdwSye6e+eXc3/zRUxb7zyItBZcvVp2tIJpx5kXJb+CoUQakbfWEQQFk4xrOF0R56+8Y="
-    - secure: "pfGVPucvgcgcefhtzAVeejlpOtn4TnAqJSTM8mJkBl36KdU9P7hMuD3czQ4drWgzZ373/VT5UVhLu/HNsdbW0YBTeqPKJ4YNjqVVLytI8xT7y2Lw9l+I7o93j98TMgAoo8nVRmp/E4D6yutbKK1eddrcmf899R1iJbw8v8d1Ht8="
 
 before_install:
-  - sudo pip install --upgrade cpp-coveralls
-  - sudo pip install ${MONGODB_ORCHESTRATION_REPO}
```

Comment:
I can't see where mongo orchestration is installed through in the new code.

Reply:
See ef72444d5239f43792a942cd9938b89bb4fbde1c. These were merely artifacts from when Hannes was using Mongo Orchestration locally to spin up test environments ([PHPC-128](https://jira.mongodb.org/browse/PHPC-128)). I found nothing in our current `.travis.yml` files that actually launched Mongo Orchestration, so this all appears to be dead code.